mathc.c

```diff
@@ -1328,15 +1329,15 @@ mfloat_t *vec3_clamp(mfloat_t *result, mfloat_t *v0, mfloat_t *v1, mfloat_t *v2)
 	return result;
 }
 
-mfloat_t *vec3_cross(mfloat_t *result, mfloat_t *v0, mfloat_t *v1)
+mfloat_t *vec3_cross(mfloat_t *result, const mfloat_t *v0, const mfloat_t *v1)
 {
 	mfloat_t cross[VEC3_SIZE];
```

Comment:
In this specific case the local variable cross[] is not nessary and the results can directly be written into result[]